src/stdlib/globals.md

```diff
@@ -112,13 +112,13 @@ By making use of the following special type checks, especially in generic contex
 
 ```ts
 function add<T>(a: T, b: T): T {
-  return a + b // addition if numeric, string concatenation if a string
+  return a + b // addition if numeric, string concatenation if string
 }
 
 function add<T>(a: T, b: T): T {
-  if (isString<T>()) { // eliminated if T is not a string
+  if (isString<T>()) { // eliminated if T is string
```

Comment:
Might be a little unclear, but the comments here refer to the respective branch, which in this case is eliminated if `T` is **not** a string. Then the `else` branch will be the only one compiled.